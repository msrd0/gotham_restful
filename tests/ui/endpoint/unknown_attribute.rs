#[macro_use]
extern crate gotham_restful;

#[derive(Resource)]
#[resource(read_all)]
struct FooResource;

#[read_all(pineapple = "on pizza")]
fn read_all() {}

fn main() {}
