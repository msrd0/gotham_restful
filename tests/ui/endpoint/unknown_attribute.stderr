error: Unknown attribute
 --> tests/ui/endpoint/unknown_attribute.rs:8:12
  |
8 | #[read_all(pineapple = "on pizza")]
  |            ^^^^^^^^^

error[E0412]: cannot find type `read_all___gotham_restful_endpoint` in this scope
 --> tests/ui/endpoint/unknown_attribute.rs:5:12
  |
5 | #[resource(read_all)]
  |            ^^^^^^^^ not found in this scope
